src/app/pages/ErrorPage/index.jsx

```diff
@@ -54,18 +54,17 @@ const ErrorMain = ({ status }) => {
         messaging={messaging}
         themeColor={themeColor}
       />
-      <ErrorPageComponent
-        {...messaging}
-        dir={dir}
-        script={script}
-        service={service}
-      />
+      <ErrorMain {...messaging} dir={dir} script={script} service={service} />
     </>
   );
 };
 
-ErrorMain.propTypes = {
-  status: number.isRequired,
+ErrorPage.propTypes = {
+  errorCode: number,
```

Comment:
I think this prop should be required. Since we never want to render this page without the errorCode having a value.

Reply:
There is a fallback to a 500 if we don't provide one but I've made it required anyway 
https://github.com/bbc/simorgh/pull/5714/commits/ca02bbfe24378e97aeb1582147188f99fc1927e9